docs/readme.rst

```diff
@@ -1,58 +1,20 @@
 :orphan:
 
-Documentation starter pack
-==========================
+==============================
+Working with the documentation
+==============================
 
-See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/>`_ guide for instructions on how to get started with Sphinx documentation.
+**Note:** This page is for documentation contributors. It assumes that this
+repository has been enabled for doc builds as described in file ``setup.rst``.
 
-Then go through the following sections to use this starter pack to set up your docs repository.
+There are make targets defined in the ``Makefile`` file that set up your local
```

Comment:
We have multiple `Makefile`s in this repository, so we should be more specific here. i.e.: `docs/Makefile`

Reply:
I will make it more explicit but I cannot give a path because the doc stuff can apply to a non-code-repository (`Makefile` is in the root) and a code repository may conceivably use a directory other than `docs`.

Reply:
Do you mean that there's a use case in which documentation sources are distributed without rest of this repository?

Reply:
So I'm taking a generic approach here. I could have provided a path **for this current repository** but the docs starter pack upon which all this is based can be used within a standalone doc repository where there *is no* docs directory. It's all in the root.